src/Generator/Generators/CSharp/CSharpSources.cs

```diff
@@ -645,7 +645,9 @@ private void GenerateClassInternalsField(LayoutField field, Class @class)
 
             var safeIdentifier = SafeIdentifier(field.Name);
```

Comment:
It looks like you don't need "safeIdentifier" any more in this method. If so, delete it.